packages/zefyr/lib/src/widgets/editable_text_block.dart

```
@@ -80,29 +78,29 @@ class EditableTextBlock extends StatelessWidget {
     return children.toList(growable: false);
   }
 
-  Widget _buildLeading(
+  Widget? _buildLeading(
       BuildContext context, LineNode node, int index, int count) {
-    final theme = ZefyrTheme.of(context);
+    final theme = ZefyrTheme.of(context)!;
     final block = node.style.get(NotusAttribute.block);
     if (block == NotusAttribute.block.numberList) {
       return _NumberPoint(
         index: index,
         count: count,
-        style: theme.paragraph.style,
+        style: theme.paragraph?.style,
         width: 32.0,
         padding: 8.0,
       );
     } else if (block == NotusAttribute.block.bulletList) {
       return _BulletPoint(
-        style: theme.paragraph.style.copyWith(fontWeight: FontWeight.bold),
+        style: theme.paragraph?.style.copyWith(fontWeight: FontWeight.bold),
         width: 32,
       );
     } else if (block == NotusAttribute.block.code) {
       return _NumberPoint(
         index: index,
         count: count,
-        style: theme.code.style
-            .copyWith(color: theme.code.style.color.withOpacity(0.4)),
+        style: theme.code?.style
+            .copyWith(color: theme.code?.style.color?.withOpacity(0.4)),
```

I think we should make all theming accessors above non-nullable. If a theme does not provide any of these we should fail early.

---

Similarly for all code that accesses theme data below as well.

---

So why don't we make `ZefyrThemeData` variables non-nullable? This way instead of runtime errors, users will get analyzer and compile errors.

---

I agree, let's make all fields in `ZefyrThemeData` non-nullable.